app/src/main/java/com/google/droidjump/LevelsFragment.java

```diff
@@ -29,20 +27,22 @@
 import android.widget.ImageButton;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentActivity;
 import com.google.droidjump.models.LevelManager;
+import com.google.droidjump.models.NavigationHelper;
 import java.util.Objects;
 
 /**
  * Displays Levels Screen.
  */
 public class LevelsFragment extends Fragment {
-    private MainActivity activity;
+    private FragmentActivity activity;
```

Comment:
why is this change necessary?

Reply:
I don't want depend on class name. In this case you can change activity class in any you want. (In all files we have FragmentActivity). FragmentActivity is a parent class of MainActivity.

Reply:
Discussed offline and decided to revert change.